src/sql/src/statements/statement.rs

```diff
@@ -89,6 +91,37 @@ pub enum Statement {
     ShowVariables(ShowVariables),
 }
 
+impl Display for Statement {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Statement::Query(s) => s.inner.fmt(f),
+            Statement::Insert(s) => s.inner.fmt(f),
+            Statement::Delete(s) => s.inner.fmt(f),
+            Statement::CreateTable(s) => s.fmt(f),
+            Statement::CreateExternalTable(s) => s.fmt(f),
+            Statement::CreateTableLike(s) => s.fmt(f),
+            Statement::DropTable(s) => s.fmt(f),
+            Statement::DropDatabase(s) => s.fmt(f),
+            Statement::CreateDatabase(s) => s.fmt(f),
+            Statement::Alter(s) => s.fmt(f),
+            Statement::ShowDatabases(s) => s.fmt(f),
+            Statement::ShowTables(s) => s.fmt(f),
+            Statement::ShowColumns(s) => s.fmt(f),
+            Statement::ShowIndex(s) => s.fmt(f),
+            Statement::ShowCreateTable(s) => s.fmt(f),
+            Statement::DescribeTable(s) => s.fmt(f),
+            Statement::Explain(s) => s.inner.fmt(f),
+            Statement::Copy(s) => s.fmt(f),
+            Statement::Tql(s) => s.fmt(f),
+            Statement::TruncateTable(s) => s.fmt(f),
+            Statement::SetVariables(s) => s.fmt(f),
+            Statement::ShowVariables(s) => s.fmt(f),
+            Statement::ShowCharset(s) => s.fmt(f),
+            Statement::ShowCollation(s) => s.fmt(f),
```

Comment:
For these two statements, only format the `ShowKind` seems incorrect. We should format like:

```
SHOW CHARSET <kind>;
SHOW COLLATION <kind>;
```

instead.

Reply:
Do we have some tests cover these cases? It seems we should not have such issues if test coverage is enough.